ts/torch_handler/base_handler.py

```diff
@@ -78,38 +89,66 @@ def initialize(self, context):
         self.manifest = context.manifest
 
         model_dir = properties.get("model_dir")
-        model_pt_path = None
+        self.model_pt_path = None
         if "serializedFile" in self.manifest["model"]:
             serialized_file = self.manifest["model"]["serializedFile"]
-            model_pt_path = os.path.join(model_dir, serialized_file)
+            self.model_pt_path = os.path.join(model_dir, serialized_file)
 
         # model def file
         model_file = self.manifest["model"].get("modelFile", "")
 
         if model_file:
             logger.debug("Loading eager model")
-            self.model = self._load_pickled_model(model_dir, model_file, model_pt_path)
+            self.model = self._load_pickled_model(
+                model_dir, model_file, self.model_pt_path
+            )
             self.model.to(self.device)
-        else:
-            logger.debug("Loading torchscript model")
-            if not os.path.isfile(model_pt_path):
-                raise RuntimeError("Missing the model.pt file")
+            self.model.eval()
+
+        elif self.model_pt_path.endswith(".pt"):
+            logger.info("Loading torchscript model")
+            self.model = self._load_torchscript_model(self.model_pt_path)
+            self.model.eval()
+
+        elif self.model_pt_path.endswith(".ts") and tensorrt_enabled:
```

Comment:
TorchTensort module could also be saved as .pt extension as well. https://github.com/pytorch/TensorRT/blob/master/examples/fx/fx2trt_example.py#L138, We might be able to use the model_config to specify the runtime. It may be an extra step for user but it can keep the code and logic clear for user and avoid any potential error while deploying the model.

Reply:
Ok yeah that makes sense I'll move tensor RT out of packaging and into the model config

Reply:
I'll work on tensorRT in another PR